### ISSUE TITLE
Migrate Mercury to puppeteer

### ISSUE THREAD

**Issue by danielepolencic** [uasabi/noderosso#3]
Current code in https://github.com/uasabi/noderosso/tree/master/nodes/mercury

**TL;DR:** The current code uses JSDOM to render the page and extract the article content with [mozilla/readability](https://github.com/mozilla/readability). Instead of JSDOM, the code should use puppeteer.

## Requirements

- [x] You should run puppeteer sever locally with `docker run -p 3000:3000 browserless/chrome`.
- [x] You should connect to puppeteer with `const browser = await puppeteer.connect({browserWSEndpoint: 'ws://localhost:3000',});`
- [x] You should add the [mozilla/readability](https://github.com/mozilla/readability) script after the page is loaded in puppeteer. 
- [x] You should launch the [mozilla/readability](https://github.com/mozilla/readability) function and collect the extract data with `var article = new Readability(document).parse();`.
- [x] You should disconnect from puppeteer when completed.

## Gotchas

- Bazel doesn't make it easy to read files from the `node_modules` folder. Please see the note in the [README of the project](https://github.com/uasabi/noderosso#reading-files-from-the-node_modules-folder)
